src/providers/arcgisrest/qgsafsprovider.h

```diff
@@ -89,6 +88,11 @@ class QgsAfsProvider : public QgsVectorDataProvider
     QVariantMap mRendererDataMap;
     QVariantList mLabelingDataList;
     QgsStringMap mRequestHeaders;
+
+    /**
+     * Clears cache
+    */
```

Comment:
FYI, when overriding virtual methods, they will inherit the docs from the parent. Adding specific docs will overwrite the parent docs. Therefore, only add docstrings if they have more information than the generic one in the parent.

Reply:
Thanks.
In this case it's fine I guess since the providers make different things to reload the data.